rebasehelper/application.py

```diff
@@ -538,7 +538,7 @@ def build_source_packages(self):
             spec = self.spec_file if version == 'old' else self.rebase_spec_file
             package_name = spec.get_package_name()
             package_version = spec.get_version()
-            package_full_version = spec.get_full_version()
+            package_full_version = spec.get_version()
```

Comment:
I don't think there is a reason to have both `package_version` and `package_full_version` if they are the same.